tests/unit/core/components/TestInstallButton.js

```diff
@@ -205,6 +205,38 @@ describe(__filename, () => {
     expect(button).toHaveProp('disabled', true);
   });
 
+  it('renders a switch button if useButton is false', () => {
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        hasAddonManager: true,
+      }),
```

Comment:
I think you meant:

```js
const root = render({
  addon: createInternalAddon({
    ...fakeAddon,
  }),
  hasAddonManager: true,
});
```

And in the other test.

Reply:
Yeah, bad assumptions in copypasta, fixing up now.